### DISCUSSION TITLE
Sortable with list of Objects

### DISCUSSION THREAD

**Discussion by octokatherine** [clauderic/dnd-kit#318] [Q&A]
I tried to use the Sortable library with an array of objects and ran into some issues. Is this not supported?

All the examples in the docs are simple arrays of numbers or strings.
https://docs.dndkit.com/presets/sortable

Thanks!

**Comment by clauderic** [comment]
Hey @octokatherine, `SortableContext` accepts `items` that adhere to the following shape: `string[]` or `{id: string}[]`

This means you can also pass it arrays of objects as long as they have an `id` property that is a unique string for each item.
